prereise/gather/griddata/hifld/const.py

```diff
@@ -408,3 +408,10 @@
 
 substation_load_share = 0.5
 demand_per_person = 2.01e-3
+
+contiguous_us_bounds = {
+    "east": -66,
+    "north": 50,
+    "south": 25,
+    "west": -125,
+}
```

Comment:
Do have a reference for this rough bound of contiguous U.S. as a spheral rectangle? If so, put it down in the doc string would be helpful.

Reply:
Do we have the shape file of the borders (not the state one, the country)?

Reply:
No reference for these bounds, I manually looked at lat/lon coordinates of the edges of Washington, Florida, and Maine. This approach still may include some lines that cross into Canada or Mexico within this bounding box, but it effectively takes care of Alaska, Hawaii, and Puerto Rico.